larq_zoo/sota/quicknet.py

```diff
@@ -9,7 +9,8 @@
 from larq_zoo.core.model_factory import ModelFactory
 
 
-def LCEFirstLayer(filters: int, x: tf.Tensor) -> tf.Tensor:
+def stem_module(filters: int, x: tf.Tensor) -> tf.Tensor:
```

Comment:
The stem module name is already taken in https://arxiv.org/pdf/1908.05858.pdf - please rename

Reply:
How about `expansion_module`?

Reply:
I would prefer `stem_module` as well. STEM is a pretty general name for the first few input layer also used in https://arxiv.org/pdf/1812.01187.pdf and I think since it is defined in the `quicknet` file it is clear that it relates to quicknet and not to an other model.

Reply:
`ScienceTechnologyEngineeringMath_module` maybe?